common/src/state/chats.rs

```diff
@@ -217,12 +220,9 @@ impl Chats {
     }
 
     /// returns the UUID of the message being replied to by the active chat
-    pub fn get_replying_to(&self) -> Option<Uuid> {
-        self.active.and_then(|id| {
-            self.all
-                .get(&id)
-                .and_then(|chat| chat.replying_to.as_ref().map(|msg| msg.id()))
-        })
+    pub fn get_replying_to(&self) -> Option<&raygun::Message> {
```

Comment:
i'd prefer cloning the `raygun::Message` so that it isn't necessary to keep `state.read()` in scope.  otherwise the PR looks good.

Reply:
actually this should go in ChatData, not State. my bad